### PR TITLE
Local json

### PR DESCRIPTION
**Pull request by gidden**
this is the cymetric PR in a trio of dbtypes-related PRs. It finds `dbtypes.js` locally. This is still broken w.r.t. to the `dbtypes.js` that cyclus will install because as suggested, that dbtypes.js will have its name and id columns removed.

A fourth PR follows that will at least fix the state of affairs for the *pre 1.3* dbtypes.

### CODE REVIEW

**Comment by opotowsky** [comment]
I'm getting the same failure even with installing cyclus via https://github.com/cyclus/cyclus/pull/1184

**Comment by gidden** [comment]
Try changing dbtypes.js to dbtypes.js.old in genapi.py

**Comment by opotowsky** [comment]
Ahhhh, genapi.py in cymetric is still configured to look for columns that aren't there any longer.

**Comment by gidden** [comment]
Look at the updatets pr, it should have that fix and this one

**Comment by gidden** [comment]
@rwcarlsen, this also should be good to go. It simply reads the local version of the dbtypes.js file. Because this does not include 1.3 types, everything builds fine and passes.

**Comment by gidden** [comment]
ok, ready to go

**Comment by rwcarlsen** [comment]
Need to delete obsolete Find[bla] cmake stuff and copy in the new FindCyclus.

File: CMakeLists.txt [hunk 1]
```diff
@@ -45,45 +50,10 @@ execute_process(COMMAND "${CYTHON_EXECUTABLE}" "-V"
...
-FIND_PACKAGE(LAPACK REQUIRED)
-set(LIBS ${LIBS} ${LAPACK_LIBRARIES})
-MESSAGE("\tFound LAPACK Libraries: ${LAPACK_LIBRARIES}")
-
-# Find HDF5
-FIND_PACKAGE(HDF5 REQUIRED)
-ADD_DEFINITIONS(${HDF5_DEFINITIONS})
-set(LIBS ${LIBS} ${HDF5_LIBRARIES})
-SET(CYMETRIC_INCLUDE_DIR "${HDF5_INCLUDE_DIRS}" ${CYMETRIC_INCLUDE_DIR})
-
```
Comment (rwcarlsen) [inline CMakeLists.txt]: I think all these are still required - we just needed to move the cyclus find to the top.

File: CMakeLists.txt [hunk 2]
```diff
@@ -45,45 +50,10 @@ execute_process(COMMAND "${CYTHON_EXECUTABLE}" "-V"
...
-# find coin and link to it
-FIND_PACKAGE(COIN REQUIRED)
-SET(CYMETRIC_INCLUDE_DIR ${CYMETRIC_INCLUDE_DIR} ${COIN_INCLUDE_DIRS})
-set(LIBS ${LIBS} ${COIN_LIBRARIES})
-
-# Find Cyclus, it is kind of a big deal
-find_package(Cyclus REQUIRED)
-SET(CYMETRIC_INCLUDE_DIR ${CYMETRIC_INCLUDE_DIR} ${CYCLUS_CORE_INCLUDE_DIR})
-SET(LIBS ${LIBS} ${CYCLUS_CORE_LIBRARIES})
-
```
Comment (rwcarlsen) [inline CMakeLists.txt]: still required too.

**Comment by gidden** [comment]
i'm just going to close this and open the docker pr

**Pull request closed without merging**